src/test/java/edu/cornell/kfs/rass/batch/xml/fixture/RassXmlAgencyEntryFixture.java

```diff
@@ -15,8 +15,8 @@
     LIMITED_LTD("555", "Limited", "Limited Ltd.", "C", "468", "Limited Ltd.", "USA", false),
     UNLIMITED_LTD("556", "Unlimited", "Unlimited Ltd.", "C", "555", "Unlimited Ltd.", "USA", false),
     FIJI_DOT("975", "Fiji_DoT", "Fiji Department of Treasury", "G", "", "Fiji Treasury Department", "Foreign", false),
-    DoS_LONG_DESC("579", "SM", "United States Department of Something with Very Long Text", "F", "",
-            "Some United States Government Department with a Significantly Long Name", "USA", false),
+    DoS_LONG_DESC("579", "SM", "United States Department of Something with Very...", "F", "",
+            "Some United States Government Department with a...", "USA", false),
```

Comment:
The RassXmlAgencyEntryFixture enum already has some helper methods to auto-truncate the full name and common name when generating the Agency objects. If you can modify those truncation methods to include the new ellipsis handling, then you can update the "DoS_LONG_DESC" enum constant to contain the un-truncated full name and common name.

Reply:
I switched to updating getTruncatedFullName and getTruncatedCommonName